### PR TITLE
Feature/1884 deposit withdraw v2

### LINKED ISSUE #1884: Deposit/Withdraw Modal Version 2 - Design Specification
Withdraw:Depoist

Note:
- Replace Approval Component with new approval button
- Replace Min. Amount info component with a new deposit/withdraw-form structure
    - Show bridge fee 

**Figma**:
[Design Specification](https://www.figma.com/file/R056tSYMjvfwaFidEBItYy/Widgets-V2?node-id=3832%3A241)
[Playground](https://www.figma.com/proto/aTivxuHmp2l6Sn8nho1Ngu/Playground?page-id=1892%3A54309&node-id=1892%3A54420&viewport=-474%2C222%2C0.10631878674030304&scaling=min-zoom)

### PR DESCRIPTION
**Pull request by Mi-Lan**
closes #1884  #1893

### CODE REVIEW

**Comment by Mi-Lan** [comment]
@KadenZipfel fucked up commit history here cause i started working on it off of Omen bridge pr before it was merged 😥  Should I do something?

File: app/src/components/modal/modal_deposit_withdraw/index.tsx
Comment (gem1no) [inline app/src/components/modal/modal_deposit_withdraw/index.tsx:77]: i learned from the very best, so this value should be passed through `props.theme.fonts.defaultLineHeight`

File: app/src/components/modal/modal_deposit_withdraw/index.tsx
Comment (hexyls) [inline app/src/components/modal/modal_deposit_withdraw/index.tsx:483]: after setting allowance the green Approved message shows up just fine but is quickly hidden again. I think we might need two seperate allowanceStates for omn and dai. Then for example we can update `daiWalletAllowance` and to be true if `daiAllowanceState` is `ButtonStates.finished` so that the Approved message can continue to be shown after setting allowance.

**Review by hexyls — CHANGES_REQUESTED** [review]
Looking good man! We can add some extra allowance state then I think this is good to goooo

File: app/src/components/modal/modal_deposit_withdraw/index.tsx
Comment (pimato) [reply app/src/components/modal/modal_deposit_withdraw/index.tsx:483]: great feedback and can confirm regarding the quickly hidden thing.

**Comment by Mi-Lan** [comment]
@hexyls @pimato Fixed it! Heres video of how it is now....https://www.dropbox.com/s/dpp6uh1ulxuwr4i/Screen%20Recording%202021-05-13%20at%2018.11.51.mov?dl=0

**Comment by pimato** [comment]
Active button has the wrong border, currently:
#E8EAF6
Bildschirmfoto 2021-05-13 um 21 22 22


Should be:
#3F51B5 

Bildschirmfoto 2021-05-13 um 21 22 52

**Comment by pimato** [comment]
pending approval button has active border button state:
Bildschirmfoto 2021-05-13 um 22 11 01

should have disabled border color (like on the right):
Bildschirmfoto 2021-05-13 um 22 11 44

**Comment by pimato** [comment]
I am just trying to withdraw some GEN, and it looks like the icon has no hover message:
Bildschirmfoto 2021-05-17 um 13 33 33

**Comment by pimato** [comment]
It looks like after a successful withdraw on xDai, a mainnet claim triggered an error on the console:
```gnosis-conditional-exchange::Services::CPKService Error trying to claim Dai tokens from xDai bridge Cannot read property 'map' of null```

**Review by KadenZipfel — COMMENTED** [review]
My main concern with this is that when we go to add more tokens, we'll have to do some rewriting since many things are manually set when it comes to OMN/DAI. Ideally we have a xdaiRelay token list on the dxTokenRegistry, and we read from that token list to automatically read token properties to render the same things without having to add any code. Depending on priorities it wouldn't be too bad to ship with this to start with, but would save a lot of trouble in the future if we can get it right from the start

**Comment by Mi-Lan** [comment]
> My main concern with this is that when we go to add more tokens, we'll have to do some rewriting since many things are manually set when it comes to OMN/DAI. Ideally we have a xdaiRelay token list on the dxTokenRegistry, and we read from that token list to automatically read token properties to render the same things without having to add any code. Depending on priorities it wouldn't be too bad to ship with this to start with, but would save a lot of trouble in the future if we can get it right from the start

@KadenZipfel Yeah I agree if we have plans to add more tokens into it. I wasn't aware of that...I was thinking that this will be solely used for the xDai and Omen token. But it would then also require changes on the UI side?

File: app/src/components/button/button_stateful/index.tsx
Comment (gem1no) [inline app/src/components/button/button_stateful/index.tsx:65]: if you wanted to use the style library you could replace line 65 with 
`font-weight: ${props => props.state === ButtonStates.idle && props.theme.buttonSecondary.weight};`

File: app/src/components/button/button_stateful/index.tsx
Comment (gem1no) [inline app/src/components/button/button_stateful/index.tsx:43]: I assume there's no way to get these to show without `!important`? generally speaking makes code harder to maintain

File: app/src/components/common/tooltip/img/IconInfo.tsx
Comment (gem1no) [inline app/src/components/common/tooltip/img/IconInfo.tsx:12]: here

File: app/src/theme/overrides/react_tooltip_styles_override.tsx
Comment (gem1no) [inline app/src/theme/overrides/react_tooltip_styles_override.tsx:78]: and these

**Review by gem1no — CHANGES_REQUESTED** [review]
see comments! just small changes, maybe the `!important` css must stay as is, not sure.

File: app/src/components/button/button_stateful/index.tsx
Comment (Mi-Lan) [reply app/src/components/button/button_stateful/index.tsx:43]: Yeah in general it isn't good to use important but in this case I think it makes sense since this class is applied conditionally I don't see other way without repeating some code to increase specificity of that class. Do you have some suggestion?

File: app/src/theme/overrides/react_tooltip_styles_override.tsx
Comment (Mi-Lan) [reply app/src/theme/overrides/react_tooltip_styles_override.tsx:78]: Only green code is written to this pr here I just put ";" cause it was screaming some small error at me . Dunno how else would this be written though?

File: app/src/components/common/tooltip/img/IconInfo.tsx
Comment (Mi-Lan) [reply app/src/components/common/tooltip/img/IconInfo.tsx:12]: Good catch not needed here!

**Comment by Mi-Lan** [comment]
@gem1no Included changes in the guild token lis pr #1921 and answered comments

**Pull request merged**
